{
  "uid" : "dd2eef221dc81409",
  "name" : "Verify that the 'All Specialists' link leads an authorized user \n                         to the correct page after clicking",
  "fullName" : "tests.specialists_page_test.TestSpecialistPageLinks#test_sp_03_03_verify_all_specialists_link_leads_authorized_user_to_the_correct_page",
  "historyId" : "a700a0fce4c3e2666e471508243072ed",
  "time" : {
    "start" : 1728694652217,
    "stop" : 1728694652217,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56460c45a02a <unknown>\n#1 0x56460c1405e0 <unknown>\n#2 0x56460c1387eb <unknown>\n#3 0x56460c128a09 <unknown>\n#4 0x56460c12a744 <unknown>\n#5 0x56460c128cbd <unknown>\n#6 0x56460c12852f <unknown>\n#7 0x56460c128422 <unknown>\n#8 0x56460c126493 <unknown>\n#9 0x56460c126ada <unknown>\n#10 0x56460c142f41 <unknown>\n#11 0x56460c1d48c5 <unknown>\n#12 0x56460c1b4b22 <unknown>\n#13 0x56460c1d3d7d <unknown>\n#14 0x56460c1b48c3 <unknown>\n#15 0x56460c1826b3 <unknown>\n#16 0x56460c18368e <unknown>\n#17 0x56460c424a2b <unknown>\n#18 0x56460c4289b1 <unknown>\n#19 0x56460c411225 <unknown>\n#20 0x56460c429532 <unknown>\n#21 0x56460c3f638f <unknown>\n#22 0x56460c448f28 <unknown>\n#23 0x56460c4490f3 <unknown>\n#24 0x56460c458e7c <unknown>\n#25 0x7fe172294ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0c626a8f8042f5d1ab1f94f4a3f4b8bd\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f8f5b306fc0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x56460c4490f3 \\\\u003Cunknown>\\\\n#24 0x56460c458e7c \\\\u003Cunknown>\\\\n#25 0x7fe172294ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x56460c45a02a <unknown>\nE       #1 0x56460c1405e0 <unknown>\nE       #2 0x56460c1387eb <unknown>\nE       #3 0x56460c128a09 <unknown>\nE       #4 0x56460c12a744 <unknown>\nE       #5 0x56460c128cbd <unknown>\nE       #6 0x56460c12852f <unknown>\nE       #7 0x56460c128422 <unknown>\nE       #8 0x56460c126493 <unknown>\nE       #9 0x56460c126ada <unknown>\nE       #10 0x56460c142f41 <unknown>\nE       #11 0x56460c1d48c5 <unknown>\nE       #12 0x56460c1b4b22 <unknown>\nE       #13 0x56460c1d3d7d <unknown>\nE       #14 0x56460c1b48c3 <unknown>\nE       #15 0x56460c1826b3 <unknown>\nE       #16 0x56460c18368e <unknown>\nE       #17 0x56460c424a2b <unknown>\nE       #18 0x56460c4289b1 <unknown>\nE       #19 0x56460c411225 <unknown>\nE       #20 0x56460c429532 <unknown>\nE       #21 0x56460c3f638f <unknown>\nE       #22 0x56460c448f28 <unknown>\nE       #23 0x56460c4490f3 <unknown>\nE       #24 0x56460c458e7c <unknown>\nE       #25 0x7fe172294ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728694652993,
      "stop" : 1728694653628,
      "duration" : 635
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56460c45a02a <unknown>\n#1 0x56460c1405e0 <unknown>\n#2 0x56460c1387eb <unknown>\n#3 0x56460c128a09 <unknown>\n#4 0x56460c12a744 <unknown>\n#5 0x56460c128cbd <unknown>\n#6 0x56460c12852f <unknown>\n#7 0x56460c128422 <unknown>\n#8 0x56460c126493 <unknown>\n#9 0x56460c126ada <unknown>\n#10 0x56460c142f41 <unknown>\n#11 0x56460c1d48c5 <unknown>\n#12 0x56460c1b4b22 <unknown>\n#13 0x56460c1d3d7d <unknown>\n#14 0x56460c1b48c3 <unknown>\n#15 0x56460c1826b3 <unknown>\n#16 0x56460c18368e <unknown>\n#17 0x56460c424a2b <unknown>\n#18 0x56460c4289b1 <unknown>\n#19 0x56460c411225 <unknown>\n#20 0x56460c429532 <unknown>\n#21 0x56460c3f638f <unknown>\n#22 0x56460c448f28 <unknown>\n#23 0x56460c4490f3 <unknown>\n#24 0x56460c458e7c <unknown>\n#25 0x7fe172294ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728694652994,
        "stop" : 1728694653627,
        "duration" : 633
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56460c45a02a <unknown>\n#1 0x56460c1405e0 <unknown>\n#2 0x56460c1387eb <unknown>\n#3 0x56460c128a09 <unknown>\n#4 0x56460c12a744 <unknown>\n#5 0x56460c128cbd <unknown>\n#6 0x56460c12852f <unknown>\n#7 0x56460c128422 <unknown>\n#8 0x56460c126493 <unknown>\n#9 0x56460c126ada <unknown>\n#10 0x56460c142f41 <unknown>\n#11 0x56460c1d48c5 <unknown>\n#12 0x56460c1b4b22 <unknown>\n#13 0x56460c1d3d7d <unknown>\n#14 0x56460c1b48c3 <unknown>\n#15 0x56460c1826b3 <unknown>\n#16 0x56460c18368e <unknown>\n#17 0x56460c424a2b <unknown>\n#18 0x56460c4289b1 <unknown>\n#19 0x56460c411225 <unknown>\n#20 0x56460c429532 <unknown>\n#21 0x56460c3f638f <unknown>\n#22 0x56460c448f28 <unknown>\n#23 0x56460c4490f3 <unknown>\n#24 0x56460c458e7c <unknown>\n#25 0x7fe172294ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"0c626a8f8042f5d1ab1f94f4a3f4b8bd\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728694482412,
      "stop" : 1728694482412,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728694652218,
      "stop" : 1728694652993,
      "duration" : 775
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728694653663,
      "stop" : 1728694653725,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Specialists Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "specialists_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestSpecialistsPage"
  }, {
    "name" : "host",
    "value" : "fv-az1245-423"
  }, {
    "name" : "thread",
    "value" : "2340-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.specialists_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 10,
        "skipped" : 7,
        "passed" : 362,
        "unknown" : 0,
        "total" : 379
      },
      "items" : [ {
        "uid" : "b4ae4aef52f986c4",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/149//#testresult/b4ae4aef52f986c4",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x555ea535802a <unknown>\n#1 0x555ea503e5e0 <unknown>\n#2 0x555ea50367eb <unknown>\n#3 0x555ea5026a09 <unknown>\n#4 0x555ea5028744 <unknown>\n#5 0x555ea5026cbd <unknown>\n#6 0x555ea502652f <unknown>\n#7 0x555ea5026422 <unknown>\n#8 0x555ea5024493 <unknown>\n#9 0x555ea5024ada <unknown>\n#10 0x555ea5040f41 <unknown>\n#11 0x555ea50d28c5 <unknown>\n#12 0x555ea50b2b22 <unknown>\n#13 0x555ea50d1d7d <unknown>\n#14 0x555ea50b28c3 <unknown>\n#15 0x555ea50806b3 <unknown>\n#16 0x555ea508168e <unknown>\n#17 0x555ea5322a2b <unknown>\n#18 0x555ea53269b1 <unknown>\n#19 0x555ea530f225 <unknown>\n#20 0x555ea5327532 <unknown>\n#21 0x555ea52f438f <unknown>\n#22 0x555ea5346f28 <unknown>\n#23 0x555ea53470f3 <unknown>\n#24 0x555ea5356e7c <unknown>\n#25 0x7f032b094ac3 <unknown>",
        "time" : {
          "start" : 1728608256577,
          "stop" : 1728608256577,
          "duration" : 0
        }
      }, {
        "uid" : "cfc14e0ad9035ab",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/cfc14e0ad9035ab",
        "status" : "passed",
        "time" : {
          "start" : 1728557379184,
          "stop" : 1728557381772,
          "duration" : 2588
        }
      }, {
        "uid" : "845f98090e1cb496",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/845f98090e1cb496",
        "status" : "passed",
        "time" : {
          "start" : 1728554247782,
          "stop" : 1728554249514,
          "duration" : 1732
        }
      }, {
        "uid" : "ffede2ec7395fa06",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/148//#testresult/ffede2ec7395fa06",
        "status" : "passed",
        "time" : {
          "start" : 1728522253577,
          "stop" : 1728522256183,
          "duration" : 2606
        }
      }, {
        "uid" : "83a692830879e66a",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/320//#testresult/83a692830879e66a",
        "status" : "passed",
        "time" : {
          "start" : 1728470648990,
          "stop" : 1728470651149,
          "duration" : 2159
        }
      }, {
        "uid" : "6e06a005d8fc3165",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/319//#testresult/6e06a005d8fc3165",
        "status" : "passed",
        "time" : {
          "start" : 1728468075911,
          "stop" : 1728468078094,
          "duration" : 2183
        }
      }, {
        "uid" : "3e5ec2413bf97c46",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/147//#testresult/3e5ec2413bf97c46",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728435895183,
          "stop" : 1728435895183,
          "duration" : 0
        }
      }, {
        "uid" : "bcd8fbf16a025d42",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/318//#testresult/bcd8fbf16a025d42",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728378104989,
          "stop" : 1728378104989,
          "duration" : 0
        }
      }, {
        "uid" : "b68371c337bb006a",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/317//#testresult/b68371c337bb006a",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728375313666,
          "stop" : 1728375313666,
          "duration" : 0
        }
      }, {
        "uid" : "4591a35b2dabbf23",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/316//#testresult/4591a35b2dabbf23",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728375195229,
          "stop" : 1728375195229,
          "duration" : 0
        }
      }, {
        "uid" : "e3b3caf9d5edf397",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/146//#testresult/e3b3caf9d5edf397",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728349283960,
          "stop" : 1728349283960,
          "duration" : 0
        }
      }, {
        "uid" : "b15e765fae7cfbc7",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/315//#testresult/b15e765fae7cfbc7",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728293033999,
          "stop" : 1728293033999,
          "duration" : 0
        }
      }, {
        "uid" : "4ed010ac27703f60",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/314//#testresult/4ed010ac27703f60",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728290083571,
          "stop" : 1728290083571,
          "duration" : 0
        }
      }, {
        "uid" : "8ecdcc936f557ee5",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/145//#testresult/8ecdcc936f557ee5",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/groups",
        "time" : {
          "start" : 1728263636513,
          "stop" : 1728263636513,
          "duration" : 0
        }
      }, {
        "uid" : "cc3b762720b5fd7b",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/313//#testresult/cc3b762720b5fd7b",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/groups",
        "time" : {
          "start" : 1728226806486,
          "stop" : 1728226806486,
          "duration" : 0
        }
      }, {
        "uid" : "4af79f3de1077428",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/312//#testresult/4af79f3de1077428",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/groups",
        "time" : {
          "start" : 1728221902949,
          "stop" : 1728221902949,
          "duration" : 0
        }
      }, {
        "uid" : "7ccbd9b8a6f23f94",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/144//#testresult/7ccbd9b8a6f23f94",
        "status" : "passed",
        "time" : {
          "start" : 1728177081591,
          "stop" : 1728177083312,
          "duration" : 1721
        }
      }, {
        "uid" : "57d2e6a62f8a6027",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/311//#testresult/57d2e6a62f8a6027",
        "status" : "passed",
        "time" : {
          "start" : 1728117714237,
          "stop" : 1728117716006,
          "duration" : 1769
        }
      }, {
        "uid" : "f8bcff2abc3854da",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/310//#testresult/f8bcff2abc3854da",
        "status" : "passed",
        "time" : {
          "start" : 1728109092910,
          "stop" : 1728109094645,
          "duration" : 1735
        }
      }, {
        "uid" : "4390d7d25f45b205",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/143//#testresult/4390d7d25f45b205",
        "status" : "passed",
        "time" : {
          "start" : 1728090350514,
          "stop" : 1728090352225,
          "duration" : 1711
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "dd2eef221dc81409.json",
  "parameterValues" : [ ]
}